<launch>
   <arg name="battery_runtime" default="30"/> 

   <node pkg="rbx2_utils" name="battery_simulator" type="battery_simulator.py" output="screen" >
     <param name="rate" value="5" />
     <param name="battery_runtime" value="$(arg battery_runtime)" />
   </node>
</launch>
